packages/workbox-precaching/utils/printInstallDetails.mjs

```diff
@@ -7,57 +7,53 @@
 */
 
 import {logger} from 'workbox-core/_private/logger.mjs';
+
 import '../_version.mjs';
 
 /**
  * @param {string} groupTitle
- * @param {Array<PrecacheEntry>} entries
+ * @param {Array<string>} urls
  *
  * @private
  */
-const _nestedGroup = (groupTitle, entries) => {
-  if (entries.length === 0) {
+function _nestedGroup(groupTitle, urls) {
```

Comment:
Why change away from `const`?

Reply:
Just for consistency's sake, since the other function uses `function`.